modules/openapi-generator/src/main/resources/typescript-fetch/sagaApiManager.mustache

```diff
@@ -18,15 +18,7 @@ export class Api {
 {{/apis}}
 {{/apiInfo}}
 
-    public static init(apiBasePath: string) {
-        const apiBaseConfig: ConfigurationParameters = {
-            basePath: apiBasePath,
-            credentials: "include",
-            headers: {
-                'Cache-Control': 'no-cache, no-store' // this is needed to prevent stalling issues in Chrome. Also it is a good behavior for api calls.
-            }
-        };
-
+    public static init(apiBaseConfig: ConfigurationParameters) {
```

Comment:
Is this a breaking change as the init method signature has changed?

Reply:
yes, technically this is a breaking change. It was done to provide more flexibility to the application. It only affect those using the saga&record mode, which almost certainly means only us for now. No project using the default typescriptfetch mode is affected by this.

Reply:
So in other words, I did not touch at all the code for the base typescriptfetch. Only the sage&record mode is tweaked with this pull request.
Just to be perfectly clear: The original typescriptfetch remains now totally unaffected with zero modifications and this was also the case when we did the previous huge pull request that was merged.